src/Parts/InstallParams.php

```diff
@@ -9,7 +9,7 @@
 class InstallParams
 {
     /**
-     * @var \Ragnarok\Fenrir\Enums\Parts\Scopes[]
+     * @var Scopes[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)